benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -366,8 +367,9 @@ private TestResult generateLoad(List<BenchmarkProducer> producers, long testDura
                         rateLimiter.acquire();
 
                         final long sendTime = System.nanoTime();
+                        String key = randomKeys[(int) sendTime % randomKeys.length];
```

Comment:
Would it simpler to make this a UUID?

Reply:
I.e. `UUID.randomUUID()`

Reply:
I prefer going with small keys so that the overhead on small size payload will be smaller